chain/types/signature.go

```diff
@@ -55,10 +55,8 @@ func (s *Signature) TypeCode() int {
 		return IKTSecp256k1
 	case KTBLS:
 		return IKTBLS
-	case "":
-		return IKTUnknown
 	default:
-		panic("unsupported signature type")
+		return IKTUnknown
```

Comment:
I'd also log an error here - how doas a signature like that make it into the system?